tsl/src/compression/compression.c

```diff
@@ -323,14 +346,165 @@ compress_chunk(Oid in_table, Oid out_table, const ColumnCompressionInfo **column
 
 	TupleDesc in_desc = RelationGetDescr(in_rel);
 	TupleDesc out_desc = RelationGetDescr(out_rel);
+	in_rel_index_oids = RelationGetIndexList(in_rel);
+	int i = 0;
+	/* We create index on the compressed chunk based on ColumnCompressionInfo,
+	 * So, before calling row compressor we need to sort the rows as per ColumnCompressionInfo.
+	 * Cost of sorting can be mitigated if we find and existing BTREE index defined for
```

Comment:
typo `and existing` should be `an existing`

Reply:
done..